testsuite/integration-arquillian/servers/cache-server/infinispan/infinispan/pom.xml

```diff
@@ -31,7 +31,8 @@
     
     <properties>
         <cache.server>infinispan</cache.server>
-        
+        <!--  CrossDC tests are failing with ISPN version 13.0.10, these tests will be updated in future therefore we leave this version on ISPN 12  -->
```

Comment:
I wonder how much use the test results are when we run them with a different ISPN version. What about disabling them completely?

Reply:
As far as I know, there is no replacement for these tests at the moment, so I would say we need to leave this running for now. I also raised this at the team meeting and the output was to leave them running with the older version.

Regarding the removal of workarounds, this is a little bit more complicated. Currently, CrossDC tests basically run two different versions of ISPN clustered together. The legacy store is using embedded ISPN and it is possible to connect it to a remote ISPN cluster. Currently (not introduced by this PR), embedded ISPN is using version 13 (from Wildfly/Quarkus dist), but the external nodes are using version 12.1.7 (from this^^^^ pom file). 

Maybe we should investigate what workarounds are touching external and what embedded ISPN and not remove those touching external ISPN to leave crossDC tests stable? WDYT?

Reply:
I assume this is OK for now, eventually this will "bite" us once the internal Infinispan has an incompatible binary protocol with the external one. I added this to the "second half" of this issue #13962.

Consider it resolved.